store/orm/orm_test.go

```diff
@@ -264,7 +267,11 @@ func TestORM_UnscopedJobRunsWithStatus_Happy(t *testing.T) {
 		test := tt
 		t.Run(test.name, func(t *testing.T) {
 
-			pending, err := store.UnscopedJobRunsWithStatus(test.statuses...)
+			var pending []*models.JobRun
+			err := store.UnscopedJobRunsWithStatus(func(jr *models.JobRun) error {
+				pending = append(pending, jr)
```

Comment:
Optional: I found it helpful to make a `cltest.AllJobs`. Perhaps a `cltest.AllJobRunsWithStatus(...)` could come in handy later down the road.